packages/quantic/force-app/examples/main/lwc/exampleQuanticRefineModalContent/exampleQuanticRefineModalContent.js-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>52.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+      <target>lightning__RecordPage</target>
+      <target>lightning__AppPage</target>
+      <target>lightning__HomePage</target>
+      <target>lightningCommunity__Page</target>
+      <target>lightningCommunity__Default</target>
+  </targets>
```

Comment:
Do we really want to expose in all of these targets? keep in mind this adds this example component in all the page builder for these environment

Reply:
probably only the community page is needed here true.